BCStudentSoftwareDevTeam/celts
# List view 243

This PR fixes issue #243 

## Summary: 
An issue was occurring in which the SLC proposal form was not properly validating that an instructor was actually added to the course. This was causing courses to be created that would not have an instructor and would not show up under the faculty's list of proposals. To fix this we added a validation check that will count the `<td>` tags and ensure there are more than 5 tags, that way we know that at least one instructor has been added to the course. If there are less than 5 we block the next page until an instructor is added.

## To test:
If you create a course and the instructor is not added attempt to hit the next button and you will see it is blocked. If you then proceed to add at least one instructor you will see it will not allow you to proceed.
